Automattic/wp-calypso
# Persist inline help when it is re-opened

#### Changes proposed in this Pull Request

* Fixes the issue of the checklist inline help disappearing, when the inline help is closed. 

**Before:**

![giphy (3)](https://user-images.githubusercontent.com/1269602/55144440-df841480-5166-11e9-9891-290d818bda1c.gif)



**After:**

![giphy (2)](https://user-images.githubusercontent.com/1269602/55143799-810a6680-5165-11e9-812d-71e9357c9ff9.gif)

#### Testing instructions

1. Create a site in the Business segment
2. In the Checklist page, click the `Do it` button to be navigated to the task page. Notice that the inline help opens for that task opens as a checklist prompt.
<img width="368" alt="Screenshot 2019-03-28 at 2 30 53 PM" src="https://user-images.githubusercontent.com/1269602/55144111-202f5e00-5166-11e9-80ad-147cdb35fe1a.png">
3. Close the inline help and re-open it, verify that the inline checklist prompt that task persists. 
4. Navigate away from the task page and open the inline help. Verify that the inline help is now no longer a checklist prompt. 

<img width="1007" alt="Screenshot 2019-03-28 at 2 32 00 PM" src="https://user-images.githubusercontent.com/1269602/55144190-448b3a80-5166-11e9-8e64-a467b72f7603.png">

**Testing Note**

From the checklist page, navigate to the task page a few times (by clicking the `Do it` button) and ensure that the task page does not go into a reload loop. I noticed that as an issue before but have fixed it.
